app/src/main/java/com/example/poste/activities/EditFolderActivity_v2.java

```diff
@@ -103,5 +102,6 @@ public void onFailure(Call<ResponseBody> call, Throwable t) {
     private void sendToDashboard() {
         Intent newIntent = new Intent(EditFolderActivity_v2.this, DashboardActivity.class);
         startActivity(newIntent);
+        finish();
```

Comment:
We may be able to just call finish() without creating and starting a new intent; this would return the user to where they were before. While this is likely the dashboard anyway, I think it could be beneficial to prevent stacking up of more activity frames